Tests/MonitoringTests/MetricHitExpectation.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+@testable import PillarboxMonitoring
+
+private struct _MetricHitExpectation<Data>: MetricHitExpectation where Data: Encodable {
+    let eventName: EventName
+    private let evaluate: (MetricPayload<Data>) -> Void
+
+    fileprivate init(eventName: EventName, evaluate: @escaping (MetricPayload<Data>) -> Void) {
+        self.eventName = eventName
+        self.evaluate = evaluate
+    }
```

Comment:
```suggestion
    init(eventName: EventName, evaluate: @escaping (MetricPayload<Data>) -> Void) {
        self.eventName = eventName
        self.evaluate = evaluate
    }
```

Unlike `ComScoreHitExpectation` and `CommandersActHitExpectation` this object is private, so we don't need a `fileprivate` initializer.

Reply:
Remove superfluous scope.